RctGcs/droneSimulator.py

```diff
@@ -1051,11 +1002,7 @@ def __connection_handler(self, connection, id):
 def main():
     parser = argparse.ArgumentParser(
         description='Radio Collar Tracker Payload Control Simulator')
-    parser.add_argument('--port', type=int, default=9000)
-    parser.add_argument('--protocol', type=str,
-                        choices=['udp', 'tcp'], required=True)
-    parser.add_argument('--target', type=str, default='127.0.0.1',
-                        help='Target IP Address.  Use 255.255.255.255 for broadcast, 127.0.0.1 for local')
+    parser.add_argument('--spec', type=str, required=True)
     parser.add_argument('--clients', type=int, default=1)
```

Comment:
new usage not reflected in README

Reply:
fixed in 5b5bffc8e2b96fb41c03d3cdfdc81f50e7a32920